engine/src/test/java/com/agiletec/aps/system/services/i18n/I18nManagerWrapperIntegrationTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2023-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package com.agiletec.aps.system.services.i18n;
+
+import com.agiletec.aps.BaseTestCase;
+import com.agiletec.aps.system.services.i18n.wrapper.I18nLabelBuilder;
+import com.agiletec.aps.util.ApsProperties;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class I18nManagerWrapperIntegrationTest extends BaseTestCase {
```

Comment:
Convert this in unit test, in this situation INtegration test is not useful and will slow down the overall time of tests

Reply:
Converted to unit test